features/README.md

```diff
@@ -59,7 +60,42 @@ SPS001_Basic-spatial-structure-for-buildings.feature
 
 #### Mandatory content
 `.feature` files:
-- must start with a tag to the functional part. See [Functional parts](./Functional-parts.md)
+- must include one and only one of these three tags to classify the validation category:
+    - `@implementer-agreement`
```

Comment:
```suggestion
- must include one and only one of these tags to classify the validation category:
    - `@critical`
    - `@implementer-agreement`
```

Right?

Reply:
Is this used only for the version prerequisite?

Reply:
> Is this used only for the version prerequisite?

So far, yes. See also
https://github.com/buildingSMART/ifc-gherkin-rules/blob/development/features/IFC101_Only-official-ifc-versions-allowed.feature